Services/TripService.cs

```diff
@@ -0,0 +1,95 @@
+using TravelTrack_API.Models;
+
+namespace TravelTrack_API.Services;
+
+public static class TripService
```

Comment:
Same here, remove static from any services.

Reply:
static references removed, dependency injection implemented (not yet with DTOs, context, mapper, etc.)